lib/item_repository.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'item'
+
+class ItemRepository
+  def all
+    sql = 'SELECT * FROM items'
```

Comment:
You might want to think about how to limit the number of lines which come back here - in small projects this is ok but if you had a million rows this could be a problem!